html/apps/signup/js/signup.js

```diff
@@ -81,10 +81,38 @@ $(document).ready(function()
         if(total){
             alert("One or more fields are wrong / empty. Please correct");
         } else{
-            alert("Success, send to Database");
-            //TODO: Send to Database
-        }
-    })
+            var mail = $('#email').val();
+            var pwd = $('#pass').val();
+            alert("Success, send to Database" + "\n" + "Mail: " + mail + "\n" + "Pass: " + pwd);
+            /*var posting = $.post("SignUpServlet",
+            {
+                email: mail, 
+                pass: pwd 
+            });
+
+            posting.done(function(data){
+                alert("success");
+                var result = $.parseJSON(data);
+                alert(result);
+            });*/
+            var jqxhr = $.post( "/api/signup.json", { email: mail, pass: pwd }, function() {
+                alert( "success" );
+            })
```

Comment:
```javascript
var jqxhr = $.post( "/api/signup.json", { email: mail, pass: pwd }, function(data) {
    console.log(data);
}, "json" );
```
Writing code like this will accept receiving json responses and avoid triggering the `.fail()`